### PR TITLE
matejcik/prodtest 23 12

### PR DESCRIPTION
**Pull request by matejcik**
this adds prodtest to the changelog checker and adds the production-signed prodtest header.

DO NOT MERGE by clicking the button. we need to retain commit a3d2ff6f863fb627a43793d131d5f26703bcfe5e which is tagged `prodtest/23-12-18`

### CODE REVIEW

File: core/embed/prodtest/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
 
+## 0.2.4 [20th December 2023]
+
+### Added
+
+- [T2B1] `SEC READ` to read out value of SEC counter.
+- [T2B1] Check certificate chain upon `CERTDEV READ`, to block bad Optiga signatures
+  from being written to device.q
```
Comment (mmilata) [inline core/embed/prodtest/CHANGELOG.md:8]: typo?
```suggestion
  from being written to device.
```

**Review by mmilata — APPROVED** [review]
lgtm, image matches

File: core/embed/prodtest/CHANGELOG.md (hunk 1 above)
Comment (matejcik) [reply core/embed/prodtest/CHANGELOG.md]: well spotted
fixed in force-push

**Comment by matejcik** [comment]
ah crap, the main branch had changed from underneath us :(

please merge this via a merge commit @prusnak

**Comment by prusnak** [comment]
> please merge this via a merge commit @prusnak

Done

**Pull request merged**